SimPEG/PF/Magnetics.py

```diff
@@ -773,7 +910,7 @@ def get_T_mat(Xn, Yn, Zn, rxLoc):
 
      """
 
-    eps = 1e-10  # add a small value to the locations to avoid /0
+    eps = 1e-8  # add a small value to the locations to avoid /0
```

Comment:
can this be a kwarg instead? users likely won't change it, but still, it might be more clear if it is an input that can be changed